centaur/src/main/resources/standardTestCases/cmd_whitespace_mixed_same_line.test

```diff
@@ -0,0 +1,12 @@
+name: cmd_whitespace_mixed_same_line
+testFormat: workflowfailure
+
+files {
+  workflow: wdl_draft3/cmd_whitespace/mixed_same_line.wdl
+}
+
+metadata {
+    status: Failed
+    "failures.0.message": "Workflow input processing failed"
+    "failures.0.causedBy.0.message": "Cannot mix leading whitespace characters in command: [\" \", \"\\t\"]"
```

Comment:
We shouldn't need to use centaur integration tests for these, IMO the womtool validation tests are good for validating these "womification time errors"

Reply:
Good tip, thanks.

The womification test harness was not playing well with the exceptions thrown directly in the code, so I took the opportunity to refactor them out of existence.